core/modules/core/include/klepsydra/core/object_pool_publisher.h

```diff
@@ -100,6 +101,7 @@ class ObjectPoolPublisher : public Publisher<T>
                                  this->_publicationStats.name);
                     this->_publicationStats._totalEventAllocations++;
                     newEvent = std::make_shared<T>();
+                    this->_publicationStats._totalEventAllocations++;
```

Comment:
This line should be removed (it is duplicated in the line 102)

Reply:
Done